stisim/multisim.py

```diff
@@ -0,0 +1,1235 @@
+import numpy as np
```

Comment:
Maybe we can remove this an use ss.MultiSim?

Reply:
oh yes, that sounds good!